ducktape/command_line/config.py

```diff
@@ -38,7 +38,7 @@ class ConsoleConfig(object):
     RESULTS_ROOT_DIRECTORY = "./results"
 
     SESSION_LOG_FORMATTER = '[%(levelname)s:%(asctime)s]: %(message)s'
-    TEST_LOG_FORMATTER = '[%(levelname)-5s - %(asctime)s - %(module)s - %(funcName)s - lineno:%(lineno)s]: %(message)s'
+    TEST_LOG_FORMATTER = '[%(levelname)-5s %(asctime)s - %(module)s:%(lineno)s %(funcName)s]: %(message)s\033[0m'
```

Comment:
That control character removes any previous control codes (i.e., colors in this case).
The colors control codes are added with logging level name.